### PR TITLE
Updates read only and rollover sections for ISM

### LINKED ISSUE #573: [DOC] ISM documentation - rollover skip, read_only action
Can you help adding documentation for rollover skip and read_only action, thanks.

1. read_only action

On this [line](https://github.com/opensearch-project/documentation-website/blob/main/_im-plugin/ism/policies.md?plain=1#L130), change it to 

Set the index setting `index.blocks.write` to true for a managed index. Notice this block doesn't mean index cannot refresh.

2. rollover skip

After this [line](https://github.com/opensearch-project/documentation-website/blob/main/_im-plugin/ism/policies.md?plain=1#L259), add

If you need to skip the rollover action, you can set this index setting `index.plugins.index_state_management.rollover_skip` to true. For example, if you see this error message "Missing alias or not the write index...", you can set the skip rollover setting, and call retry API to skip rollover action.

### PR DESCRIPTION
**Pull request by ariamarble**
### Description

Updates read only and rollover sections for ISM

### Issues Resolved

fixes #573 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _im-plugin/ism/policies.md [hunk 1]
```diff
@@ -259,6 +261,8 @@ Rolls an alias over to a new index when the managed index meets one of the rollo
 
 **Important**: ISM checks the conditions for operations on **every execution of the policy** based on the **set interval**, _not_ continuously. The rollover will be performed if the value **has reached** or _exceeded_ the configured limit **when the check is performed**. For example with `min_size` configured to a value of 100GiB, ISM might check the index at 99 GiB and not perform the rollover. However, if the index has grown past the limit (e.g. 105GiB) by the next check, the operation is performed.
```
Comment (vagimeli) [inline _im-plugin/ism/policies.md:262]:
```suggestion
**Important**: ISM checks the conditions for operations on **every execution of the policy** based on the **set interval**, _not_ continuously. The rollover will be performed if the value **has reached** or _has exceeded_ the configured limit **when the check is performed**. For example with `min_size` configured to a value of 100GiB, ISM might check the index at 99 GiB and not perform the rollover. However, if the index has grown past the limit (e.g., 105GiB) by the next check, the operation is performed.
```

**Review by vagimeli — APPROVED** [review]
@ariamarble Minor edits.

File: _im-plugin/ism/policies.md (hunk 1 above)
Comment (Naarcha-AWS) [inline _im-plugin/ism/policies.md:262]:
```suggestion
ISM checks the conditions for operations on **every execution of the policy** based on the **set interval**, _not_ continuously. The rollover will be performed if the value **has reached** or _exceeded_ the configured limit **when the check is performed**. For example with `min_size` configured to a value of 100GiB, ISM might check the index at 99 GiB and not perform the rollover. However, if the index has grown past the limit (e.g. 105GiB) by the next check, the operation is performed.
{: .note}
```

File: _im-plugin/ism/policies.md [hunk 2]
```diff
@@ -257,7 +259,9 @@ Deletes a managed index.
 
 Rolls an alias over to a new index when the managed index meets one of the rollover conditions.
 
-**Important**: ISM checks the conditions for operations on **every execution of the policy** based on the **set interval**, _not_ continuously. The rollover will be performed if the value **has reached** or _exceeded_ the configured limit **when the check is performed**. For example with `min_size` configured to a value of 100GiB, ISM might check the index at 99 GiB and not perform the rollover. However, if the index has grown past the limit (e.g. 105GiB) by the next check, the operation is performed.
+**Important**: ISM checks the conditions for operations on **every execution of the policy** based on the **set interval**, _not_ continuously. The rollover will be performed if the value **has reached** or _has exceeded_ the configured limit **when the check is performed**. For example with `min_size` configured to a value of 100GiB, ISM might check the index at 99 GiB and not perform the rollover. However, if the index has grown past the limit (e.g., 105GiB) by the next check, the operation is performed.
```
Comment (ariamarble) [inline _im-plugin/ism/policies.md:262]:
```suggestion
ISM checks the conditions for operations on **every execution of the policy** based on the **set interval**, _not_ continuously. The rollover will be performed if the value **has reached** or _has exceeded_ the configured limit **when the check is performed**. For example with `min_size` configured to a value of 100GiB, ISM might check the index at 99 GiB and not perform the rollover. However, if the index has grown past the limit (e.g., 105GiB) by the next check, the operation is performed.
```

**Pull request merged**